wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -299,23 +313,27 @@ As a response, the API provides you with a wallet `id` used in subsequent reques
 }
 ```
 
-You have just created your first wallet. Information about this wallet can be retrieved using the `GET /api/v1/wallets/{walletId}` endpoint as follow:
+You have just created your first wallet. Information about this wallet can be retrieved using the [`GET /api/v1/wallets/{walletId}`](#tag/Wallets%2Fpaths%2F~1api~1v1~1wallets~1{walletId}%2Fget)
+endpoint as follows:
 
 ```
 curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}} \
-     -H "Accept: application/json; charset=utf-8"              \
-     --cacert ./scripts/tls-files/ca.crt                       \
+     -H "Accept: application/json; charset=utf-8" \
+     --cacert ./scripts/tls-files/ca.crt
 ```
 
-Receiving Money
----------------
+Receiving Ada
+-------------
 
-To receive money from other users you should provide your address. This address can be obtained from an account. Each wallet contains at least one account, you can think of account as a pocket inside of your wallet. Besides, you can view all existing accounts of a wallet by using the `GET /api/v1/wallets/{{walletId}}/accounts` endpoint as follow:
+To receive _Ada_ from other users you should provide your address. This address can be obtained
```

Comment:
Genuine question (I don't know the answer): usually currencies (e.g. USD, EUR, etc) are spelled out in caps-lock. Are you aware of any convention in this space we should abide?

Reply:
I believe that this refers to the alpha-3 code from the [ISO-4217](https://www.iso.org/iso-4217-currency-codes.html) standard:

> The alphabetic code is based on another ISO standard, ISO 3166, which lists the codes for country names. The first two letters of the ISO 4217 three-letter code are the same as the code for the country name, and where possible the third letter corresponds to the first letter of the currency name.
>
>For example:
>
>    - The US dollar is represented as USD - the US coming from the ISO 3166 country code and the D for dollar.
>    - The Swiss franc is represented by CHF - the CH being the code for Switzerland in the ISO 3166 code and F for franc.

Even though *ADA* isn't yet in the standard, it'd make sense to align with that I suppose. Good call ^.^